.mvn/extensions.xml

```diff
@@ -24,7 +24,7 @@
   <extension>
     <groupId>com.gradle</groupId>
     <artifactId>develocity-maven-extension</artifactId>
-    <version>1.23</version>
+    <version>1.22.2</version>
```

Comment:
Should we downgrade from 1.23?

Reply:
Yes, ge.apache.org and develocity.apache.org are on version 2024.2.7, which is only compatible with extensions <= 1.22.2. 2024.3 is required for extension 1.23.

The migration to the new develocity.apache.org server should lead to faster updates to help prevent these scenarios where a (presumably) automated PR updates the extension before the server is ready to accept it.

Reply:
Thanks for the explanation.  We can also disable auto-update for this dependency.